src/Service/BucketService.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Service;
+
+use League\Flysystem\Filesystem;
+
+class BucketService
+{
+    private $filesystem;
+
+    public function __construct(Filesystem $filesystem) {
+        $this->filesystem = $filesystem;
+    }
+
+    public function uploadFile ($file, $dir) {
+        $filesystem = $this->filesystem;
+        $path = "/$dir/".$file->getClientOriginalName();
```

Comment:
The path should probably be generated to be something other than the original file name as otherwise 1) different comments can't have the same filename uploaded, 2) the extension might not match the filename correctly (possible security issue)